docs/crio.conf.d.5.md

```diff
@@ -0,0 +1,25 @@
+% crio.conf.d(5)
+
+# NAME
+crio.conf.d - directory for drop-in configuration files
+
+# DESCRIPTION
+Additionally to configuration in crio.conf(5) CRI-O allows to drop configuration
+snippets into the crio.conf.d directory. The default directory is /etc/crio/crio.conf.d/.
+The path can be change via CRIO's **--config-dir** command line option.
+
+# CONFIGURATION PRECEDENCE
+When existing, the main configuration file (/etc/crio/crio.conf by default) is
```

Comment:
suggest "When existing," -> "When it exists,"

Reply:
done